addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -90,3 +90,17 @@ def test_notify_reply_to_computation_mc(self):
                     "%s %s" % (self.user_employee_c2.company_id.name, test_record.name),
                     "%s@%s" % (self.alias_catchall, self.alias_domain)))
             )
+
+        # Test3: get company from record (company_id field)
+        new_company = self.env['res.company'].sudo().create({'name': 'ELIT'})
+        test_record = self.env['mail.test.multi.company'].create({
```

Comment:
Is it different from Test2 ?

Reply:
when multiple company are linked to the same record we default to the user's company 
if the record is linked to a single company we use that company as reply_to